portal_blocks_documentation/docs.py

```diff
@@ -59,7 +67,7 @@ def ensure_index_md(block_name: str):
 def ensure_index_extra_docs_file(block_name: str):
     extra_doc = BUILD_DIR / block_name / "docs" / "extra.md"
     if not extra_doc.exists():
-        extra_doc.write_text(f"<!-- Add extra documentation for {block_name} in this file -->")
+        extra_doc.write_text(f"_There is currently no additional information provided for this block by the community._")
```

Comment:
do we really need to show this? if there is not documentation we don't need to tell :thinking:

Reply:
I was mainly replacing this in order to visualize the include (and get around some hugo error regarding un-ended html content. 
And I think it doesn't hurt to make it more visible that the community could add additional documentation there.

Reply:
Well that does make sense, good change we can mark this as resolved